### PR TITLE
Fix TBB build issue

### PR DESCRIPTION
**Pull request by amklinv-nnl**
@mhoemmen Please review #252 first since this PR sits on top of it.

### CODE REVIEW

**Comment by mhoemmen** [comment]
@amklinv-nnl Sorry for the trouble, but would you mind if I rebased these changes atop PR #253 instead of PR #252?  #253 has less code duplication and avoids exposing namespace macros to the test files themselves.  Thanks!

File: examples/01_scale.cpp [hunk 1]
```diff
@@ -30,7 +22,7 @@ int main(int argc, char* argv[]) {
 
     // Call linalg::scale x = 2.0*x;
     std::experimental::linalg::scale(2.0, x);
-#ifdef MDSPAN_EXAMPLES_USE_EXECUTION_POLICIES
```
Comment (mhoemmen) [inline examples/01_scale.cpp:33]: The new macro is nicer; thank you! : - )

File: include/experimental/__p1673_bits/macros.hpp [hunk 2]
```diff
@@ -43,6 +43,30 @@
 #ifndef LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_MACROS_HPP_
 #define LINALG_INCLUDE_EXPERIMENTAL___P1673_BITS_MACROS_HPP_
 
+#ifdef _MSVC_LANG
```
Comment (mhoemmen) [inline include/experimental/__p1673_bits/macros.hpp:46]: This is a good idea : - )

**Comment by amklinv-nnl** [comment]
> @amklinv-nnl Sorry for the trouble, but would you mind if I rebased these changes atop PR #253 instead of PR #252? #253 has less code duplication and avoids exposing namespace macros to the test files themselves. Thanks!

Please rebase/resolve these PRs in whatever way makes you happy :-)

File: include/experimental/__p1673_bits/macros.hpp (hunk 2 above)
Comment (amklinv-nnl) [reply include/experimental/__p1673_bits/macros.hpp:46]: Thanks, I stole it from Kokkos!

**Comment by mhoemmen** [comment]
This PR is superseded by PR https://github.com/kokkos/stdBLAS/pull/256 , which incorporates all of @amklinv-nnl 's changes except the new `execpolicy_mapper` overload (which would best be reserved for a subsequent PR that does an actual `std::execution::*` overload).  Thanks to @amklinv-nnl for the implementation!

**Pull request closed without merging**